README.md

```diff
@@ -22,17 +22,17 @@ go get github.com/omise/omise-go
 
 # COMPLIANCE WARNING
 
-Card data should never transit through your server. This library provides means to create
-tokens on the server side but should only be used for testing or **only if you currently
+Credit Card data should never transfer through your server. This library provides a means to create
```

Comment:
We dont use "Credit" word because there also "Debit" cards. In general case, it's usually refereed as just "Card Data"